test/pom.xml

```diff
@@ -42,6 +41,17 @@
 		</repository>
 	</repositories>
 
+	<dependencies>
```

Comment:
Tried that but it didn't work so I had to keep the dependency elements. Let me know if you want me to try something else.